model/src/main/java/org/cbioportal/model/StructVarFilterQuery.java

```diff
@@ -0,0 +1,48 @@
+package org.cbioportal.model;
+
+import org.cbioportal.model.util.Select;
+import org.springframework.lang.Nullable;
+
+import java.io.Serializable;
+
+public class StructVarFilterQuery extends BaseAlterationFilter implements Serializable {
```

Comment:
I think it is better to spell the name out as `StructuralVariantFilterQuery` for consistency. For example, we already have `StructuralVariantGeneSubQuery`.